cmd/osm-controller/osm-controller.go

```diff
@@ -79,7 +80,7 @@ var (
 )
 
 func init() {
-	flags.StringVarP(&verbosity, "verbosity", "v", "info", "Set log verbosity level")
+	flags.StringVarP(&verbosity, "verbosity", "v", constants.DefaultOSMLogLevel, "Set boot log verbosity level")
```

Comment:
If the initial logging is set using MeshConfig, why is this needed?

Reply:
^ from commit msg
```
The initial log level flag has been kept as it still mandates boot log
level up to the point configurator and the log level handler are both
up, which then meshconfig value takes over.
```